### PR TITLE
Add option to reset personal bests (#980)

### PR DESCRIPTION
**Pull request by bsamseth**
Ref. discussion https://github.com/Miodec/monkeytype/discussions/980

The new settings button:
image

After clicking it:
image

Stats page after reset:
image

### CODE REVIEW

File: static/quotes/english.json [hunk 1]
```diff
@@ -1,22 +1,10 @@
...
-      600
-    ],
-    [
-      601,
-      9999
-    ]
+    [0, 100],
+    [101, 300],
+    [301, 600],
+    [601, 9999]
```
Comment (bsamseth) [inline static/quotes/english.json:7]: This, and the other language file, was changed automatically while merging updates, so I assume it is fine to leave them in this PR even though they are unrelated formatting changes.

File: functions/index.js [hunk 2]
```diff
@@ -455,6 +455,22 @@ exports.removeSmallTestsAndQPB = functions.https.onCall(
   }
 );
 
+exports.resetPersonalBests = functions.https.onCall(
+  async (request, response) => {
+    console.log("heyheyhey");
```
Comment (0xfr34ky) [inline functions/index.js:460]: hey

Comment (Miodec) [reply functions/index.js:460]: hey hey hey!

File: static/quotes/english.json (hunk 1 above)
Comment (Miodec) [reply static/quotes/english.json:7]: Yeah this is fine

File: functions/index.js (hunk 2 above)
Comment (bsamseth) [reply functions/index.js:460]: I see I left that one in there on accident. I would have removed it, but I see it is merged already. Oh well, very happy to have it merged so quickly! Hey hey hey!

**Comment by bsamseth** [comment]
@Miodec I see this has already been pushed to the live website, but I don't think the deployment did everything it should. Trying to click the new reset button doesn't do anything and leaves this error in the console:

image

Looking through the sources I see other `new SimplePopups` definitions present, but not the one added here. Perhaps only parts of the code are deployed currently? I.e. the static index page is updated but the built js code has not?

I guess this sorts itself out as soon as you do another build update, but thought I'd let you know! 👍🏻
